development/tools/jobs/incubator/hack-showcase/github_connector_test.go

```diff
@@ -8,9 +8,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestHackShowcaseJobPresubmit(t *testing.T) {
+func TestGithubConnectorJobPresubmit(t *testing.T) {
```

Comment:
Please change for this job names of jobs -> `pre-master-github-connector` and `post-master-github-connector`.  You have already duplicated names for two job and this is incorrect in prow. Update tests and also change version of golang buildpack to mentioned in other comment.